guides/v2.1/contributor-guide/contributing_docs.md

```diff
@@ -8,26 +8,38 @@ github_link: contributor-guide/contributing_docs.md
+If you write and contribute a full topic, we will put your name (or company's name) at the top of the page, right under the title, and link it to your blog or website! We will also add your picture and a link to your Github account on the [Top recent contributors]({{ page.baseurl }}/contributor-guide/quarterly-contributors.html#top-recent-contributors) page.
 
 ## Contribution guidelines
-We use [Markdown](http://daringfireball.net/projects/markdown/){:target="\_blank"} to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html){:target="\_blank"}. We have a <a href="{{ page.baseurl }}/contributor-guide/templates/basic_template.html" target="\_blank">template</a> to make getting started easy.
+We use [Markdown](http://daringfireball.net/projects/markdown/){:target="_blank"} to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html){:target="_blank"}.
+
+You can update or add content to existing topics. We have all topics in versioned directories for 2.0, 2.1, 2.2, and onward.
 
-{% include note.html type="info" content="Refer to the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca) for detailed information about licensing. All contributors are required to submit a click-through form to agree to the terms." %}
+We provide templates to help get you started on new topics:
+
+* [General topic template]({{ page.baseurl }}/contributor-guide/templates/basic_template.html) with example formats and styles.
+* Tutorial templates with example formats and styles for step-by-step instructions, like How Tos. It includes templates for the [first introduction topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-first.html), [middle topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-middle.html) for each step, and a [final step topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-last.html).
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
+Refer to the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca) for detailed information about licensing. All contributors are required to submit a click-through form to agree to the terms.
+</div>
 
 1.  First, check [existing pull requests](https://github.com/magento/devdocs/pulls){:target=\_blank"} and make sure you are not duplicating work!
 
-1.  For large contributions or changes that include multiple files, [open an issue](https://github.com/magento/devdocs/issues){:target="\_blank"} and discuss it with us first. This may prevent duplicate or unnecessary effort.
+1.  For large contributions or changes that include multiple files, [open an issue](https://github.com/magento/devdocs/issues){:target="_blank"} and discuss it with us first. This may prevent duplicate or unnecessary effort.
```

Comment:
Apply changes to this page as noted in previously commented on pages here. I don't want to retype everything!

Reply:
Not a problem :)